client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/pattern-large-preview.tsx

```diff
@@ -123,6 +124,7 @@ const PatternLargePreview = ( {
 				) }
 			>
 				<PatternRenderer
+					key={ device }
```

Comment:
The component needs to be re-mounted to calculate the content height based on the new viewport width. This creates a flash. Any other idea to force the component to recalculate the content height?